frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -939,6 +939,8 @@ export const surveyLogic = kea<surveyLogicType>([
                                     q.question
                                 }`
                             }),
+                            `coalesce(JSONExtractString(properties, '$lib_version')) -- Library Version`,
+                            `coalesce(JSONExtractString(properties, '$lib')) -- Library`,
```

Comment:
can you move these to after the timestamp and person fields?  This is useful data to surface but i don't think it's more important than the timestamp and the person, so it should appear after those columns in the table.

Reply:
Sounds good, doing that now!